modules/ROOT/pages/intro-configuration.adoc

```diff
@@ -3,23 +3,33 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Mule 3, Spring property placeholders are often used to configure apps dynamically for the environment in which they are deployed. Mule 4 contains a built-in mechanism for this that allows you to set default values and avoid the need to learn Spring.
+In Mule 3, Spring property placeholders are often used to configure apps dynamically for the environment in which they are deployed. Mule 4 contains a built-in mechanism for this that allows you to set default values and avoid the need to learn Spring. +
 
-These properties are stored in a YAML file:
+Mule 3 supports `.properties` configuration files, while Mule 4 supports both `.yaml` and `.properties`  configuration files. The recommended approach is to use `.yaml` configuration files, because it allows the addition of type validations and autocompletion.
+
+Example of a `.yaml` configuration file:
```

Comment:
```suggestion
Following is an example of a `.yaml` configuration file:
```

Reply:
Not sure we need to use both "YAML file" and " `.yaml` " file.
Which is the most useful or appropriate or commonly understood?
Can we standardize to just one?